app/models/lighthouse/education_benefits/enrollment.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'active_support/inflector'
+
+module Lighthouse
+  module EducationBenefits
+    # The Enrollment model represents an enrollment of a veteran in an education program.
+    # This model is used to parse and manipulate the enrollment data returned from the Lighthouse API.
+    # It includes ActiveModel::Model to get some of the ActiveRecord features, such as validations and conversions,
+    # but it does not persist data to a database.
+    class Enrollment
```

Comment:
I'm used to the Django pattern, where you can create a serializer (or form) to transform/validate some data.  It doesn't appear that Rails has a non-model based serializer, so I settled on this solution of creating a model for the purpose of the data transformation but *does not* represent a database table/row.

Happy to modify my technique, if this solution isn't appropriate.

Reply:
I may be misunderstanding what you are trying to do here but, is it possible you are looking to do something like what the [post911 serializer](https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/serializers/post911_gi_bill_status_serializer.rb) does? 

Wondering the same thing for the app/models/lighthouse/education_benefits/education_benefit.rb class? Are we looking to use a serializer?

Reply:
I believe you can add custom methods to a serializer, which then will return the method's name as the attribute and the result of the method call as the value.

Reply:
Good question, Kyle.  As I understand it, Rails wants a model to pass to your serializer.

Current functionality of the code makes a service request to EVSS and then instantiates a [GiBillStatusResponse](https://github.com/department-of-veterans-affairs/vets-api/blob/VA-IIR-448-new_GIBS_serializer/lib/evss/gi_bill_status/gi_bill_status_response.rb#L43) model object, which does a lot of the lifting to divine what's going on with the response that comes back from an opaque service.  Then it's fed into a serializer to be turned into JSON.

Because the model I created just changes some key names around, it did not seem necessary to also create a separate serializer; it would essentially double the size of the PR without any benefit of functionality or readability.

I was reading and found https://api.rubyonrails.org/classes/ActiveModel/Serialization.html which might be helpful? I'm realizing that I don't need to include `ActiveModel::Model` into my objects anymore (that was from a previous experiment).  But I'm not actually sure what including `ActiveModel::Serializers` would buy me, either.  Works just fine without it.

One "Rails Convention" question that I have, that you might be able to answer, is the placement of these files.  AFAIK Rails suggests models being placed in `app/models` but maybe this is not a "Capital-M Model" and should be physically closer to the other code.  I put it in `app/models/lighthouse/benefits_education/` but it could go alongside the service in `lib/lighthouse/benefits_education/`

What do you think?

Reply:
Update: Turns out I _do_ need `ActiveModel::Model`, hah